### DISCUSSION TITLE
How can I ask gpt-pilot to respect changes I will make manually in the codes?

### DISCUSSION THREAD

**Discussion by hdmbn** [Pythagora-io/gpt-pilot#484] [Q&A]
I frequently notice that GPT-Pilot struggles to debug straightforward issues. To address this, I plan to modify certain files to rectify the problem. It's important that GPT-Pilot acknowledges these adjustments and applies its changes to new files, rather than the versions stored in its database. Is there a way to ensure that GPT-Pilot behaves in this manner?

**Comment by blarvin** [comment]
I had good results with this! 

First, I got to the end of app creation with GPT-Pilot (I'm using VS Code ext). Second, I made a lot of improvements, refactoring, restructuring, etc. Third, I restarted GPT-Pilot on the same project. It asked if I had made changes and if I wanted to keep them. I said yes. Next I prompted the new changes I wanted in the project, and re-stated the changes I ha already made. Then GPT-Pilot went to work, and new changes were good.
